### DISCUSSION TITLE
Slack invitations on the website.

### DISCUSSION THREAD

**Discussion by clcoffey** [InnerSourceCommons/InnerSourceMarketing#182] [General]
A week ago @lenucksi  mentioned that the Slack Inviter Heroku will discontinue its service to us because of a change in its business model. 

We use Heroku on the website for people that want to join our slack community.

Here are the options I found so far:
- Reach out to them and ask them if they would make an exception for a Non for Profit. Their pricing mentions a "Free and Hobby" tier for Non-commercial apps such as proof of concepts, MVPs and personal projects.
- If they say no, the lowest price they offer after that is $25/month
- Use the invite link we can generate in Slack. - This link allows for up to 400 people to join before it has to be updated.

I would appreciate people's thoughts on this issue outlining any other options that we might have.

**Comment by voborgus** [comment]
Thanks for raising this. My opinion is to try the first option, and if this will not work – go with the last option. Since the our sizes – 400 people works well for us.
For the long term we can also look on some cloud serverless invitators – providers usually have large free tier options for functions and we can have the save what we had in Heroku.

**Comment by spier** [comment]
@clcoffey I assume that we are using this approach, right?
> Use the invite link we can generate in Slack. - This link allows for up to 400 people to join before it has to be updated.

How do we know that the 400 people has been reached? i.e. who would receive an alert about this and has to create a new link?

**Reply by clcoffey** [reply]
@claredillon was keeping an eye on it. We didn't really have any alerts.

**Reply by spier** [reply]
@claredillon what's your take? Do we need to derisk this?

**Comment by claredillon** [comment]
The latest link we included didn't seem to have any mention of people limits. I think we're good. But perhaps it might be worthwhile to schedule an intermittent test (every 3 months say) just to make sure the link still works...
